### PR TITLE
Update duplicated input validation

### PR DESCRIPTION
**Pull request by rbioteau**
Only validate duplicate at same level
Remove icon from label provider has it trigger some label display issue (cut at the end)

### CODE REVIEW

**Comment by rbioteau** [comment]
Merge https://github.com/bonitasoft/bonita-studio/pull/50 first

File: bundles/plugins/org.bonitasoft.studio.contract/src-test/java/org/bonitasoft/studio/contract/ui/property/input/labelProvider/InputNameCellLabelProviderTest.java [hunk 1]
```diff
@@ -15,58 +15,29 @@
...
-    }
-
-    /**
-     * @throws java.lang.Exception
-     */
-    @After
-    public void tearDown() throws Exception {
-    }
+    @Rule
+    public RealmWithDisplay RealmWithDisplay = new RealmWithDisplay();
```
Comment (apupier) [inline bundles/plugins/org.bonitasoft.studio.contract/src-test/java/org/bonitasoft/studio/contract/ui/property/input/labelProvider/InputNameCellLabelProviderTest.java]: attribute name with an uppercase?

File: bundles/plugins/org.bonitasoft.studio.contract/src/org/bonitasoft/studio/contract/ui/property/input/labelProvider/InputNameCellLabelProvider.java [hunk 2]
```diff
@@ -35,12 +31,7 @@ public InputNameCellLabelProvider(final IPropertySourceProvider propertySourcePr
 
     @Override
     public Image getImage(final Object object) {
-        return Pics.getImage("ContractInput.gif", ContractPlugin.getDefault());
-    }
-
-    @Override
-    public StyledString getStyledText(final Object element) {
-        return new StyledString(getText(element));
+        return null;//Pics.getImage("ContractInput.gif", ContractPlugin.getDefault());
```
Comment (apupier) [inline bundles/plugins/org.bonitasoft.studio.contract/src/org/bonitasoft/studio/contract/ui/property/input/labelProvider/InputNameCellLabelProvider.java]: no mor eimage? commented code?

Comment (rbioteau) [inline bundles/plugins/org.bonitasoft.studio.contract/src/org/bonitasoft/studio/contract/ui/property/input/labelProvider/InputNameCellLabelProvider.java]: See pr comment for image removal justification

**Pull request merged**